src/core/data.jl

```diff
@@ -2721,7 +2721,15 @@ function _resolve_swithces!(data::Dict{String,<:Any})
     for bus_set in Set(values(bus_sets))
         bus_min = minimum(bus_set)
         Memento.info(_LOGGER, "merged buses $(join(bus_set, ",")) in to bus $(bus_min) based on switch status")
+        
+        # Merging a PV bus into a PQ bus results in a PV bus.
+        # Therefore, the resulting bus should always have the highest bus_type,
+        # but it should never be of type 4 (disconnected).
+        bus_type = max([data["bus"]["$i"]["bus_type"] for i in bus_set]...)
```

Comment:
```suggestion
        bus_type = maximum(data["bus"]["$i"]["bus_type"] for i in bus_set)
```

Reply:
Thanks for the review @odow and Happy New Year too! 🎉
I've always had issues with the max and maximum functions.